<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudGhostClass</class>
 <widget class="QMainWindow" name="CloudGhostClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CloudGhost</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_start">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_shading">
    <property name="title">
     <string>渲染选项</string>
    </property>
    <widget class="QMenu" name="menu_post_process">
     <property name="title">
      <string>后期处理</string>
     </property>
     <addaction name="action_pp_normal"/>
     <addaction name="action_pp_gray"/>
     <addaction name="action_pp_invers"/>
     <addaction name="action_pp_blur"/>
     <addaction name="action_pp_sharpen"/>
     <addaction name="action_pp_edge_det"/>
    </widget>
    <widget class="QMenu" name="menu_polygon_mode">
     <property name="title">
      <string>图元模式</string>
     </property>
     <addaction name="action_pm_fill"/>
     <addaction name="action_pm_line"/>
    </widget>
    <addaction name="action_color_bg"/>
    <addaction name="action_color_border"/>
    <addaction name="separator"/>
    <addaction name="menu_polygon_mode"/>
    <addaction name="separator"/>
    <addaction name="action_rb_3d"/>
    <addaction name="action_eye_delta"/>
    <addaction name="separator"/>
    <addaction name="menu_post_process"/>
   </widget>
   <widget class="QMenu" name="menushadertoy">
    <property name="title">
     <string>shadertoy</string>
    </property>
    <widget class="QMenu" name="menu_st_edit">
     <property name="title">
      <string>编辑</string>
     </property>
     <addaction name="action_st_img"/>
     <addaction name="separator"/>
     <addaction name="action_sd_buffer_a"/>
     <addaction name="action_st_buffer_b"/>
     <addaction name="action_st_buffer_c"/>
     <addaction name="action_st_buffer_d"/>
    </widget>
    <addaction name="action_st_show"/>
    <addaction name="menu_st_edit"/>
   </widget>
   <addaction name="menu_start"/>
   <addaction name="menu_shading"/>
   <addaction name="menushadertoy"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_rb_3d">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>红蓝3D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_eye_delta">
   <property name="text">
    <string>双眼视距</string>
   </property>
  </action>
  <action name="action_pp_gray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>灰度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+Q</string>
   </property>
  </action>
  <action name="action_pp_normal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>原色</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+S</string>
   </property>
  </action>
  <action name="action_color_bg">
   <property name="text">
    <string>背景颜色</string>
   </property>
  </action>
  <action name="action_color_border">
   <property name="text">
    <string>边框颜色</string>
   </property>
  </action>
  <action name="action_pp_invers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>反相</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+W</string>
   </property>
  </action>
  <action name="action_pp_blur">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>模糊</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+E</string>
   </property>
  </action>
  <action name="action_pp_sharpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>锐化</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+R</string>
   </property>
  </action>
  <action name="action_pp_edge_det">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>边缘检测</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Ctrl+T</string>
   </property>
  </action>
  <action name="action_st_show">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_sd_buffer_a">
   <property name="text">
    <string>BufferA</string>
   </property>
  </action>
  <action name="action_st_buffer_b">
   <property name="text">
    <string>BufferB</string>
   </property>
  </action>
  <action name="action_st_buffer_c">
   <property name="text">
    <string>BufferC</string>
   </property>
  </action>
  <action name="action_st_buffer_d">
   <property name="text">
    <string>BufferD</string>
   </property>
  </action>
  <action name="action_st_img">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="action_polygon_line">
   <property name="text">
    <string>_polygon_line</string>
   </property>
  </action>
  <action name="action_polygon_mode">
   <property name="text">
    <string>_polygon_mode</string>
   </property>
  </action>
  <action name="action_pm_fill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>填充模式</string>
   </property>
  </action>
  <action name="action_pm_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>线框模式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../cloudghost.qrc"/>
 </resources>
 <connections/>
</ui>
